### PR TITLE
Small improvements to Chat App tutorial

### PR DESCRIPTION
**Pull request by jenks-guo-filecoin**
- added hyperlink for libp2p early in article
- recorded and added a .gif walking through the app with four browsers, gives reader quick overview of the app and keep engagement high
- added illustration of where status indicator is
- rewarded slightly in Docker section

### CODE REVIEW

**Comment by jenks-guo-filecoin** [comment]
Hey @mishmosh 😀 should I get write access to ipfs-docs repo? Or I should just fork and submit PR like I just did.

**Comment by jenks-guo-filecoin** [comment]
Hey @TheDiscordian @mishmosh, minor improvements to the tutorial. Please kindly review and provide your approval.

I am unable to tag for reviewer for some reason. 

I enjoyed playing with the chat app a lot, it's neat!

**Comment by mishmosh** [comment]
Added a few minor comments, but defer to @TheDiscordian for approval as the original author of this tutorial.

File: docs/how-to/create-simple-chat-app.md [hunk 1]
```diff
@@ -35,9 +42,9 @@ The diagram above demonstrates what a three-user network can look like. It's wor
 
 ### Docker (optional)
 
-If you don't want to use Docker, skip to the [**WebRTC-Star**](#webrtc-star) section.
+Skip to the [**WebRTC-Star**](#webrtc-star) section, if you are not using Docker
 
-After this section we'll go over what WebRTC-Star and circuit relay do, and how to set them up. However, if you'd like to quickly roll your own kit using Docker, this example includes an image you can use. It might not be the best long-term solution, but it should be great if you want to quickly get rolling and experiment.
+After this section we'll go over what WebRTC-Star and circuit relay do, and how to set them up. However, if you'd like to quickly roll your own kit using Docker, this example includes an Docker image you can use. It might not be the best long-term solution, but it should be great if you want to quickly get rolling and experiment.
```
Comment (TheDiscordian) [inline docs/how-to/create-simple-chat-app.md:47]: s/an Docker/a Docker

File: docs/how-to/create-simple-chat-app.md [hunk 2]
```diff
@@ -35,9 +42,9 @@ The diagram above demonstrates what a three-user network can look like. It's wor
 
 ### Docker (optional)
 
-If you don't want to use Docker, skip to the [**WebRTC-Star**](#webrtc-star) section.
+Skip to the [**WebRTC-Star**](#webrtc-star) section, if you are not using Docker
```
Comment (TheDiscordian) [inline docs/how-to/create-simple-chat-app.md:45]: Looks like the "." got removed erroneously :)

**Review by TheDiscordian — CHANGES_REQUESTED** [review]
Thanks so much! Just some minor grammar changes. I think the CircuitRelay info is outdated with CircuitRelayV2, but I vastly prefer your wording over mine. I also LOVE the gifs ❤️

File: docs/how-to/create-simple-chat-app.md (hunk 1 above)
Comment (jenks-guo-filecoin) [reply docs/how-to/create-simple-chat-app.md:47]: Should be a Docker image. I will correct.

File: docs/how-to/create-simple-chat-app.md (hunk 2 above)
Comment (jenks-guo-filecoin) [reply docs/how-to/create-simple-chat-app.md:45]: ooops

File: docs/how-to/create-simple-chat-app.md [hunk 3]
```diff
@@ -35,9 +42,9 @@ The diagram above demonstrates what a three-user network can look like. It's wor
 
 ### Docker (optional)
 
-If you don't want to use Docker, skip to the [**WebRTC-Star**](#webrtc-star) section.
+Skip to the [**WebRTC-Star**](#webrtc-star) section, if you are not using Docker.
```
Comment (johnnymatthews) [inline docs/how-to/create-simple-chat-app.md:45]:
```suggestion
This step is optional. If you are not using Docker skip to the [**WebRTC-Star**](#webrtc-star) section.
```

File: docs/how-to/create-simple-chat-app.md [hunk 4]
```diff
@@ -35,9 +42,9 @@ The diagram above demonstrates what a three-user network can look like. It's wor
 
 ### Docker (optional)
```
Comment (johnnymatthews) [inline docs/how-to/create-simple-chat-app.md:43]:
```suggestion
### Docker
```

File: docs/how-to/create-simple-chat-app.md [hunk 5]
```diff
@@ -15,6 +15,9 @@ You can see the live demo [here](https://ipfs.io/ipfs/bafybeia5f2yk6td7ciroeped2
 
 Then simply open `index.html` in your web browser and you'll immediately begin automatically connecting to nodes and looking for peers!
 
+A quick video demo of the chat app: 
+![A quick video demo of the chat app with four browsers open](https://ipfs.io/ipfs/bafybeihh4zncvsazoyfnbzkjrc4a7vozp6ejhrsewfcg5p6tiqn3chccly "Quick Video Demo of the chat app")
```
[the remark this answers is not in the document]
Comment (johnnymatthews) [reply docs/how-to/create-simple-chat-app.md:19]: Agreed. The video is a great idea! But it's hard to see which screen is in focus. I'm going to delete this section and merge the rest of this PR, but please make another PR with an improved video if you have time!

**Review by johnnymatthews — APPROVED** [review]

File: docs/how-to/create-simple-chat-app.md (hunk 5 above)
Comment (johnnymatthews) [inline docs/how-to/create-simple-chat-app.md:19]:
```suggestion
```

**Review by johnnymatthews — APPROVED** [review]

**Comment by johnnymatthews** [comment]
Nice PR! I've removed the video section for now. Please take a look at mine and @mishmosh's comments to potentially improve the video, if you have time. I'm gonna merge the rest of this in now though. Thanks!

**Comment by jenks-guo-filecoin** [comment]
> Nice PR! I've removed the video section for now. Please take a look at mine and @mishmosh's comments to potentially improve the video, if you have time. I'm gonna merge the rest of this in now though. Thanks!
> 
> Edit: oh wait. Looks like @TheDiscordian has some changes requested. @TheDiscordian do you still want to see those changes, or can we merge this in now?

Thanks @johnnymatthews @mishmosh @TheDiscordian! 

On demo video: @mishmosh good feedback. I did four windows to show the peer to peer discoverability feature (best to use 3 or 3+), but it did look quite daunting and untidy, so I took your advice on it. 

Let me know if there is anything else I need to action.

**Review by johnnymatthews — APPROVED** [review]
Nice! Thanks!

**Review by TheDiscordian — APPROVED** [review]
LGTM thanks!

**Pull request merged**